### ISSUE TITLE
Add ARIA attributes for better Accessiblity

### ISSUE THREAD

**Issue by tkshill** [tkshill/Quarto#22]
# Issue Context
Our main page elements do not have the appropriate [Accessible Rich Internet Applications](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA) (ARIA) labels. This makes it harder for people with disabilities to navigate the site.

# Suggested Solution
Add as many of the accessbility [attributes](https://package.elm-lang.org/packages/mdgriffith/elm-ui/latest/Element-Region#aside) in the Elm-ui region library as are applicable. In particular:
- Add the `navigation` attribute msg to the `row` element containing the links in the `Shared.elm` view function.
- Add the `mainContent` attribute msg to the `column` containing the page.body.
- Add the `announce` attribute message to the top level column in the `viewBoard` function.

# Alternatives Considered

# Additional Resources

See [here](https://github.com/tkshill/Quarto/blob/main/README.md#run-and-install) in the read me for how to run and install the application.

See [here](https://github.com/tkshill/Quarto/blob/main/CONTRIBUTING.md#the-basics) in contributing for the basics on forking and cloning the repository.

**Comment by dominicduffin1** [comment]
Hi! I'd like to take this on. Could you confirm - is the page.body also in `Shared.elm` and `viewBoard` the one in `GamePage.elm.`?

**Comment by tkshill** [comment]
Hey @dominicduffin1 ,

I've assigned you to the issue. The `page.body` in `shared.elm` is equivalent to the `body` of the top level record returned by `view` function in `GamePage.elm`.

So this page.body in _Shared.elm_
```
view :
    { page : Document msg, toMsg : Msg -> msg }
    -> Model
    -> Document msg
view { page } _ =
            ...
            , column [ height fill, centerX ] page.body
            ...
```
is this body in _GamePage.elm_

```
view : Model -> Document Msg
view model =
    { title = "Quarto - Play"
    , body =
        [ column [ spacing 10, centerX ]
           ...
```

**Comment by tkshill** [comment]
closed by #39
